Description: Fix issues in dvi2tty manual page
Author: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>
Last-Update: 2020-05-24
Debian-Bug: #958728
Forwarded: https://github.com/mirakels/dvi2tty/pull/2

Index: texlive-bin/texk/dvi2tty/dvi2tty-src/dvi2tty.1
===================================================================
--- texlive-bin.orig/texk/dvi2tty/dvi2tty-src/dvi2tty.1	2020-09-22 22:43:00.747827109 +0200
+++ texlive-bin/texk/dvi2tty/dvi2tty-src/dvi2tty.1	2020-09-22 22:46:08.654841938 +0200
@@ -25,19 +25,19 @@
 In principle Font Changes are totally ignored, but
 .I dvi2tty
 recognizes
-a few mathematical and special symbols that can be be displayed on an
-ordinary ascii terminal, such as the '+' and '-' symbol.
+a few mathematical and special symbols that can be displayed on an
+ordinary ascii terminal, such as the '+' and '\-' symbol.
 .PP
 If the width of the output text requires more columns than fits
-in one line (c.f. the \-w option) it is broken into several lines by
+in one line (cf.\& the \-w option) it is broken into several lines by
 .I dvi2tty
 although they will be printed as one line on regular TeX output
-devices (e.g. laser printers). To show that a broken line is really
+devices (e.g., laser printers). To show that a broken line is really
 just one logical line an asterisk (``*'') in the last position
 means that the logical line is continued on the next physical
 line output by
 .I dvi2tty.
-Such a continuation line is started with a a space and an asterisk
+Such a continuation line is started with a space and an asterisk
 in the first two columns.
 .PP
 Options may be specified in the environment variable DVI2TTY.
@@ -52,11 +52,11 @@
 .TP
 .B \-p list
 Print the pages chosen by list.
-Numbers refer to TeX\-page numbers (known as \\count0).
+Numbers refer to TeX\-page numbers (known as \ecount0).
 An example of format for list is ``1,3:6,8''
 to choose pages 1, 3 through 6 and 8.
 Negative numbers can be used exactly as in TeX,
-e g \-1 comes before \-4 as in ``\-p\-1:\-4,17''.
+e.g., \-1 comes before \-4 as in ``\-p\-1:\-4,17''.
 .TP
 .B \-P list
 Like \-p except that page numbers refer to
@@ -66,7 +66,7 @@
 .B \-w n
 Specify terminal width
 .I n.
-Legal range 16\-132.
+Valid range 16\(en132.
 Default is 80. If your terminal has the
 ability to display in 132 columns it might
 be a good idea to use \-w132 and toggle the
@@ -74,7 +74,7 @@
 probably look somewhat better.
 .TP
 .B \-v
-Specify height of lines. Default value 450000. Allows to adjust linespacing.
+Specify height of lines. Default value 450000. Allows one to adjust linespacing.
 .TP
 .B \-q
 Don't pipe the output through a pager.
@@ -83,7 +83,7 @@
 .TP
 .B \-e n
 This option can be used to influence the spacing between words.
-With a negative value the number of spaces 
+With a negative value the number of spaces
 between words becomes less, with a positive value it becomes more.
 \-e\-11 seems to worked well.
 .TP
@@ -100,14 +100,14 @@
 .TP
 .B \-Fprog
 Use ``prog'' as program to pipe output into. Can be used to choose an
-alternate pager (e g ``-Fless'').
+alternate pager (e.g., ``\-Fless'').
 .TP
 .B \-t
 \\tt fonts were used (instead of cm) to produce dvi file. (screen.sty
-is a powerfull mean to do that with LaTeX).
+is a powerful mean to do that with LaTeX).
 .TP
 .B \-a
-Dvi2tty normally tries to output accented characters. With the -a option,
+Dvi2tty normally tries to output accented characters. With the \-a option,
 accented characters are output without the accent sign.
 .TP
 .B \-l
@@ -115,18 +115,18 @@
 to mark them with a form-feed character.
 .TP
 .B \-c
-Do not attempt to translate any characters (like the Scandinavion/latin1 mode)
+Do not attempt to translate any characters (like the Scandinavian/latin1 mode)
 except when running in tt-font.
 .TP
 .B \-u
 Toggle option to process certain latin1 characters. Use this if your output
-devices supports latin1 cahracters.
-Note this may interfere with -s. Best not to use -u and -s together.
+devices supports latin1 characters.
+Note this may interfere with \-s. Best not to use \-u and \-s together.
 .TP
 .B \-s
 Toggle option to process the special Scandinavian characters that on most (?)
-terminals in Scandinavia are mapped to ``{|}[\\]''.
-Note this may interfere with -u. Best not to use -u and -s together.
+terminals in Scandinavia are mapped to ``{|}[\e]''.
+Note this may interfere with \-u. Best not to use \-u and \-s together.
 .TP
 .B \-J
 Auto detect NTT JTeX, ASCII pTeX, and upTeX dvi format.
@@ -148,18 +148,17 @@
 Print the name of fonts when switching to it (and ending it). The delim
 argument is used to delimit the fontname.
 .SH FILES
-/bin/more \ \ \ \ 
-probably the default pager.
+.nr L1 \w'/bin/more \ \ \ \ '
+/bin/more\h'|\n(L1u'probably the default pager.
 .SH ENVIRONMENT
-PAGER \ \ \ \ \ \ \ \ \ \ \ \ 
-the pager to use.
+.\".nr L1 \w'DVI2TTY  '
+PAGER\h'|\n(L1u'the pager to use.
 .br
-DVI2TTY \ \ \ \ \ \ \ \ \ \ \ 
-can be set to hold command-line options.
+DVI2TTY\h'|\n(L1u'can be set to hold command-line options.
 .SH "SEE ALSO"
 TeX, dvi2ps
 .SH AUTHOR
-Original Pascal verion: Svante Lindahl, Royal Institute of Technology, Stockholm
+Original Pascal version: Svante Lindahl, Royal Institute of Technology, Stockholm
 .br
 Improved C version: Marcel Mol
 .br
